core/embed/rust/src/ui/model_tt/component/page.rs

```diff
@@ -44,6 +55,18 @@ where
         swipe
     }
 
+    fn make_content(layout: &PageLayout, content: impl FnOnce(Rect) -> T) -> T {
+        // Check if content fits on single page.
+        let mut content = content(layout.content_single_page);
+        if content.page_count() <= 1 {
+            return content;
+        }
+
+        // Reduce area to make space for scrollbar if it doesn't fit.
+        content.set_area(layout.content);
```

Comment:
Very very small nit:
```suggestion
        if content.page_count() > 1 {
            // Reduce area to make space for scrollbar if it doesn't fit.
            content.set_area(layout.content);
        }
```

Reply:
D'oh ... f4dc04a